### PR TITLE
Update project forms to use compacted JSON-LD

### PR DESCRIPTION
**Pull request by blushi**
## Description

Closes: https://github.com/regen-network/regen-registry/issues/904

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**.*

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles 
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/lib/transform.ts [hunk 1]
```diff
@@ -200,9 +200,10 @@ const getPartyFromMetadata = (
     type: metadataRole?.['@type'].includes('regen:Organization') // covers Organization or OrganizationDisplay
       ? 'ORGANIZATION' // to provide default image
       : '',
+    image: metadataRole?.['schema:image']?.['@value'],
+    address: metadataRole?.['schema:location']?.place_name || '',
```
Comment (blushi) [inline web-registry/src/lib/transform.ts:204]: not directly related but realized this was missing cc/ @mhagel

File: web-registry/src/lib/transform.ts [hunk 2]
```diff
@@ -117,7 +117,7 @@ export function buildIssuanceModalData(
         creditVintage.endDate,
       ),
       monitoringPeriods,
-      projectName: project.metadata?.['http://schema.org/name'] || '',
+      projectName: project.metadata?.['schema:name'] || '',
```
Comment (blushi) [inline web-registry/src/lib/transform.ts:120]: not directly related but when the project metadata didn't get updated to use the compacted form in here, fixed it in a few other places as well cc/ @mhagel

**Comment by victordelval** [comment]
Seem right. Nice job!

Just a question that I throw in the air in case it is of interest so that we give it one thought for a next refactor or whatever... 

would it make sense to have a utility or something that parses this type of json-LD structures to avoid using those string notations with semicolons on the keys, to improve DX? 

For example, instead of using a key string like `metadataRole?.['schema:name']` we could use something more JS friendly like `metadataRole?.schemaName` or something like this... 

It would also act as a wrapper to hide the final implementation and avoid having to change in several places if there is an update, etc.

**Review by victordelval — APPROVED** [review]

**Review by mhagel — APPROVED** [review]
Whew! That's better :)

**Comment by blushi** [comment]
> Seem right. Nice job!
> 
> Just a question that I throw in the air in case it is of interest so that we give it one thought for a next refactor or whatever...
> 
> would it make sense to have a utility or something that parses this type of json-LD structures to avoid using those string notations with semicolons on the keys, to improve DX?
> 
> For example, instead of using a key string like `metadataRole?.['schema:name']` we could use something more JS friendly like `metadataRole?.schemaName` or something like this...
> 
> It would also act as a wrapper to hide the final implementation and avoid having to change in several places if there is an update, etc.

Something to keep in mind.
The only concern that I have is that on input value change, we'll still need to convert the data back to proper JSON-LD for the SHACL validation to work or maintain 2 copies of the data (the one with nicer key names and the JSON-LD one), so I'm not sure if it's worth it.

**Comment by blushi** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**